### PR TITLE
Split the default ewasm import list out into its own module

### PR DESCRIPTION
**Pull request by jakelang**
Depends on #56 
Step 1 of `remapimports` refactor.

### CODE REVIEW

**Comment by axic** [comment]
Please don't merge this into `preset-trait`, changed the base to avoid that accident.

File: libchisel/src/imports.rs [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+                ),
+                ImportType::Function(
+                    "ethereum",
+                    "selfDestruct",
+                    FunctionType::new(vec![ValueType::I32], None),
+                ),
+            ])),
+            //.iter()
+            //.cloned()
+            //.collect()
```
Comment (axic) [inline libchisel/src/imports.rs]: Stray?

Comment (jakelang) [reply libchisel/src/imports.rs]: Fixed.

**Comment by jakelang** [comment]
@axic Is this good for merge? I have personally triple-checked the import list but we will introduce tests later.

**Comment by axic** [comment]
It was approved, just I didn't take responsibility for double checking the list 😉

**Comment by jakelang** [comment]
ok, merging

**Pull request merged**